HomeWork 4/HomeWork 4/Program.cs

```diff
@@ -0,0 +1,222 @@
+
+            ChangeOnUpperCaseSpecificChar(changeEvenArray);
+
+            int countCharUpperOfEven = CountOfUpperChar(changeEvenArray);
+
+            Console.WriteLine("Displey Even char array with Upper: ");
+
+            DisplayShowCharArray(changeEvenArray);
+
+            Console.WriteLine();
+
+            /* Start create add work with Odd number array and transfrom to char */
+
+            int[] arrayOddNumbers = new int[lengthArray];
+
+            CreateOddElementToArray(arrayOddNumbers);
+
+            char[] changeOddArray = ChangeNumberToChar(arrayOddNumbers);
+
+            ChangeOnUpperCaseSpecificChar(changeOddArray);
+
+            int countCharUpperOfOdd = CountOfUpperChar(changeOddArray);
+
+            Console.WriteLine("Displey Odd char array with Upper: ");
+
+            DisplayShowCharArray(changeOddArray);
+
+            Console.WriteLine();
+
+            /* Compare two char array */
+
+            Console.WriteLine($"array with even numbers have {countCharUpperOfEven} upper letter, array odd numbers have {countCharUpperOfOdd} upper letter");
+
+            CompareTwoValues(countCharUpperOfEven, countCharUpperOfOdd);
+        }
+
+        static void DisplayShowIntArray(int[] array)
+        {
+            foreach (var item in array)
+            {
+                Console.Write(item + " ");
+            }
+
+            Console.WriteLine();
+        }
+
+        static void DisplayShowCharArray(char[] array)
+        {
+            foreach (var item in array)
+            {
+                Console.Write(item + " ");
+            }
+
+            Console.WriteLine();
+        }
+
+        static int[] CreateRandomElement(int[] array)
+        {
+            for (int i = 0; i < array.Length; i++)
+            {
+                array[i] = new Random().Next(1, 27);
+            }
+
+            return array;
+        }
+
+        static int[] CreateEvenElementToArray(int[] array)
+        {
+            int randomNumber;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                randomNumber = new Random().Next(1, 26);
+                if (randomNumber % 2 == 0)
+                {
+                    array[i] = randomNumber;
+                }
+                else
+                {
+                     array[i] = randomNumber + 1;
+                }
+            }
+
+            return array;
+        }
+
+        static int[] CreateOddElementToArray(int[] array)
+        {
+            int randomNumber;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                randomNumber = new Random().Next(1, 26);
+                if (randomNumber % 2 == 1)
+                {
+                    array[i] = randomNumber;
+                }
+                else
+                {
+                    array[i] = randomNumber + 1;
+                }
+            }
+
+            return array;
+        }
```

Comment:
This is overengineering, we can use one method for both purposes

Reply:
I thought about it but decided to do separate what))